types/swift/AnalyticsEvent.swift

```diff
@@ -0,0 +1,36 @@
+// 
+// Copyright 2021 New Vector Ltd
```

Comment:
Copyright year is incorrect, but that links to https://github.com/matrix-org/matrix-analytics-events/issues/68

Reply:
I've added a note in that issue about `base_types` 👍